pkg/proxy/iptables/proxier.go

```diff
@@ -1495,8 +1481,8 @@ func (proxier *Proxier) syncProxyRules(reason syncReason) {
 	}
 	proxier.portsMap = replacementPortsMap
 
-	// Update healthz timestamp if it is periodic sync.
-	if proxier.healthzServer != nil && reason == syncReasonForce {
```

Comment:
Ah I think it should be fine, doesn't seem like [Load()](https://golang.org/src/sync/atomic/value.go?s=798:836#L21) will block [Store()](https://golang.org/src/sync/atomic/value.go?s=1308:1344#L38):
```
// Load returns the value set by the most recent Store.
```